src/FSharp.Stats/DistanceMetrics.fs

```diff
@@ -88,6 +88,77 @@ module DistanceMetrics =
                 if not (isNan x) then
                     dist <- dist + System.Math.Abs x
             dist
+
+        /// <summary>The [Minkowski distance](https://en.wikipedia.org/wiki/Minkowski_distance) between two vectors (ignores nan), of order `p`.</summary>
```

Comment:
If I understand this correctly, this function results in (Some nan) if a sequence contains at least one nan. 
Therefore I would suggest changing "ignores nan" to "<remark>Returns nan if sequences contain nan</remark>".

Reply:
Thanks. This is indeed incorrect in the summary. I will change it